pkg/chunk/composite_store.go

```diff
@@ -3,13 +3,18 @@ package chunk
 import (
 	"context"
 	"sort"
+	"time"
 
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/prometheus/pkg/labels"
-
-	"github.com/cortexproject/cortex/pkg/util/validation"
 )
 
+// CompositeStoreLimits helps get Limits specific to Queries for Stores in CompositeStore
+type CompositeStoreLimits interface {
```

Comment:
This seems misnamed, since it's used in `store`, `CompositeStore` and `seriesStore`.  Suggest renaming to `StoreLimits`.